Exited with an error
stdout: no U1 account found; exiting

stderr: /usr/bin/account-console:8: PyGIWarning: Accounts was imported without specifying a version first. Use gi.require_version('Accounts', '1.0') before import to ensure that the right version gets loaded.
  from gi.repository import Accounts
/usr/bin/account-console:9: PyGIWarning: Signon was imported without specifying a version first. Use gi.require_version('Signon', '1.0') before import to ensure that the right version gets loaded.
  from gi.repository import Signon

